.github/workflows/test_bundle.yml

```diff
@@ -0,0 +1,75 @@
+name: colcon-bundle Integration Test
+on:
+  pull_request:
+  schedule:
+    - cron: '0 * * * *'
+
+# Test the released and current master branch
+jobs:
+  bundle_integration_test:
+    name: "Integration Test"
+    strategy:
+      fail-fast: false
+      matrix:
+        install_source:
+          - "colcon-ros-bundle"
+          - "git+https://github.com/colcon/colcon-bundle.git@master#egg=colcon-bundle git+https://github.com/colcon/colcon-ros-bundle.git@master#egg=colcon-ros-bundle"
+    runs-on: ubuntu-latest
+    container:
+      image: rostooling/setup-ros-docker:ubuntu-xenial-ros-kinetic-ros-base-latest
+    steps:
+      - name: setup-ros-docker#7 workaround
+        run: sudo chown -R rosbuild:rosbuild "$HOME" .
+       # needed to access the vcs repos file from the workspace
+       # ensures a contribution does not break scheduled tests
+      - name: Checkout source
+        uses: actions/checkout@v2
+      - uses: ros-tooling/action-ros-ci@0.0.17
+        id: build-from-repos
+        with:
+          vcs-repo-file-url: "${{ github.workspace }}/.github/workflows/test_bundle.repos"
+          package-name: ros_tutorials navigation
+          source-ros-binary-installation: kinetic
+      - name: Install dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get -y install python3-apt python3-pip
+          sudo -H pip3 install -U setuptools==44.0.0
```

Comment:
Is it mentioned anywhere why `setuptools` is pinned to version `44.0.0`?

Reply:
I followed the [test.yml](https://github.com/colcon/colcon-bundle/blob/master/.github/workflows/test.yml#L25) usage. Should we install the latest? If so, can fix here and let's file an issue to address the unit tests.